tutorials/temporal_analysis.ipynb

```diff
@@ -0,0 +1,384 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    " # Temporal analysis with gammapy\n",
+    " \n",
+    " - PKS 2155-304 flare runs from the HESS Public data release\n",
+    " - Assume template background already made using https://docs.gammapy.org/dev/notebooks/background_model.html\n",
+    " - Do a 3D map based analysis keeping only the amplitude free\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import astropy.units as u\n",
+    "import os\n",
+    "from pathlib import Path\n",
+    "\n",
+    "from astropy.coordinates import SkyCoord, Angle\n",
+    "\n",
+    "from gammapy.data import ObservationFilter, DataStore, DataStoreObservation\n",
+    "from gammapy.irf import EffectiveAreaTable2D, EnergyDispersion2D, EnergyDependentMultiGaussPSF, Background3D\n",
+    "from gammapy.spectrum.models import PowerLaw, PowerLaw2\n",
+    "from gammapy.image.models import SkyPointSource\n",
+    "from gammapy.cube import MapEvaluator, PSFKernel, MapMaker, MapMakerObs, MapDataset\n",
+    "from gammapy.utils.fitting import Fit\n",
+    "from gammapy.cube import make_map_exposure_true_energy, make_map_background_irf\n",
+    "from gammapy.maps import WcsGeom, MapAxis, WcsNDMap, Map\n",
+    "from gammapy.cube.models import SkyModel, SkyModels, BackgroundModel\n",
+    "from gammapy.cube.exposure import _map_spectrum_weight\n",
```

Comment:
Remove _map_spectrum_weight import - it's not used, and we should never use "private" stuff from Gammapy in end-user tutorials.